api/libproject/devstart.rb

```diff
@@ -993,6 +994,64 @@ def list_clusters(cmd_name, *args)
   :fn => ->(*args) { list_clusters("list-clusters", *args) }
 })
 
+def update_cdr_version_options(cmd_name, args)
+  op = WbOptionsParser.new(cmd_name, args)
+  op.opts.dry_run = false
+  op.add_option(
+       "--file [JSON_FILE]",
+       ->(opts, v) { opts.file = v},
+       "Path to a file containing the CDR version data to use (relative to api directory).")
```

Comment:
Added defaults.

Reply:
So what is the idea here? We an a cdr version object to the array in the json file when ever we release a new cdr , then this info gets put in the cdr versions database ?

Reply:
Does this have anything to do with actually switching the default cdr the apps are using ?

Reply:
@biopete Yeah, that's the idea. 

The config for the default CDR version is presently specified by name in config, here:

https://github.com/all-of-us/workbench/blob/master/api/config/config_prod.json#L19

Though we could consider in future changing it so the default CDR is actually marked as such in the cdr_version table directly.

Reply:
Gotcha. So what is the point of the file arg ? Wouldn't it always be the same file, one for each environment and just be updated with a new object in the array?  

Essentially this is our master list of cdr releases right?  It will be committed and the update will be run on all the environments.  Is there a reason to not just check everytime something is merged with master and update the cdr version table in all the environments to match these files ? 

Should there be an active flag too ?

Reply:
Per discussion, I removed the "--file" flag from the Ruby goals. There probably isn't a good reason to update a Cloud environment with a file other than the one dictated by the Ruby.

At the moment, the code still has one file for all nonprod environments. See Slack discussion on whether we want to keep things that way.

Re: an active flag, I think we may wind up doing something like that, yes; we may also make some CDR versions accessible only to admins. But that hasn't been designed yet. For now I'm just trying to fix the issue of having to manually hack the database whenever our CDR metadata changes, and start building support for multiple CDR versions.

Reply:
Per further discussion, I changed each environment to have its own file, and changed our deploy target to automatically update CDR versions based on the latest state of that file. So you can just merge changes to a file for a given environment and in the next release it will pick up the new CDR version data. (If you want it sooner, you can run the tool manually.)